packages/arb-token-bridge-ui/src/components/common/HeaderConnectWalletButton.tsx

```diff
@@ -28,23 +22,8 @@ function NetworkButtonAndConnectButton({
     )
   }
 
-  if (chainUnsupported) {
-    return (
-      <button
-        onClick={openChainModal}
-        type="button"
-        className="arb-hover flex w-full flex-row items-center bg-[#FF494A] bg-lime-dark px-6 py-3 text-2xl font-medium text-white lg:rounded-full lg:px-4 lg:py-2 lg:text-base lg:font-normal"
-      >
-        Wrong network
-      </button>
-    )
-  }
```

Comment:
We handle wrong network in the network dropdown and this case will never be reached